newrelic/hooks/framework_graphql.py

```diff
@@ -450,15 +444,11 @@ def wrap_graphql_impl(wrapped, instance, args, kwargs):
     with GraphQLOperationTrace() as trace:
         trace.statement = graphql_statement(query)
 
-        # Handle Graphene Schemas
-        try:
-            from graphene.types.schema import Schema as GrapheneSchema
-            if isinstance(schema, GrapheneSchema):
-                trace.product = "Graphene"
-                framework = graphene_framework_details()
-                transaction.add_framework_info(name=framework[0], version=framework[1])
-        except ImportError:
-            pass
+        # Handle Schemas created from frameworks
+        if hasattr(schema, "_nr_framework"):
+            trace.product = "Graphene"
```

Comment:
Can we move this line down a line, and change the hard coded graphene to use `framework[0]`?

Reply:
Sure